File: App/CarRpc 5.21.07 PM/app/src/main/java/com/carrpc/nickd/car_rpc/CarMovements.java
package com.carrpc.nickd.car_rpc;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.*;
import android.widget.TextView;


public class CarMovements extends AppCompatActivity {
        ImageButton left = (ImageButton) findViewById(R.id.leftArr);
        ImageButton right = (ImageButton) findViewById(R.id.rightArr);
        ImageButton up = (ImageButton) findViewById(R.id.upArr);
        ImageButton down = (ImageButton) findViewById(R.id.downArr);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);
            setContentView(R.layout.activity_car_movements);
        left.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        right.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        up.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        down.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
    }
}

File: App/CarRpc 5.21.07 PM/app/src/main/java/com/carrpc/nickd/car_rpc/CarMovements.kt
package com.carrpc.nickd.car_rpc

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_car_movements.*

class CarMovements : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_car_movements)

    // Example of a call to a native method
    sample_text.text = stringFromJNI()
    }

    /**
     * A native method that is implemented by the 'native-lib' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {

        // Used to load the 'native-lib' library on application startup.
        init {
            System.loadLibrary("native-lib")
        }

    }
}
